emulators/gxemul/Portfile

```diff
@@ -13,16 +14,20 @@ consisting of processors (ARM, MIPS, Motorola 88K, PowerPC, and SuperH) \
 and surrounding hardware components such as framebuffers, interrupt \
 controllers, busses, disk controllers, and serial controllers.
 
-homepage        http://gxemul.sourceforge.net/
-master_sites    ${homepage}src/
+# There is also a sourceforge hosted subversion repo at https://sourceforge.net/projects/gxemul/
+homepage        https://gavare.se/gxemul/
+master_sites    https://gavare.se/gxemul/src/
```

Comment:
You can actually do `${homepage}/src/` here

Reply:
To avoid a double slash in the download URL, you would want it to be `${homepage}src/`.